rover/src/owr_drive_controls/CMakeLists.txt

```diff
@@ -117,6 +117,10 @@ add_executable(cmd_vel_2_joints src/cmdVelToJoints.cpp src/Drive.cpp src/crabDri
 add_dependencies(cmd_vel_2_joints owr_messages)
 target_link_libraries(cmd_vel_2_joints ${catkin_LIBRARIES})
 
+add_executable(cmd_vel_2_arm_joints src/cmdVelToArmJoints.cpp src/armControl.cpp)
+add_dependencies(cmd_vel_2_arm_joints owr_messages)
+target_link_libraries(cmd_vel_2_arm_joints ${catkin_LIBRARIES})
```

Comment:
Its not really `cmd_vel` that might be a bit confusing as thats the standard name of the ros velocity vector (fot driving)

Reply:
Renaming it to `arm_joints`